src/apps/smallbank/app/smallbank.cpp

```diff
@@ -15,6 +15,8 @@ namespace ccfapp
 {
   struct SmallBankTables
   {
+    // TODO: Update to RawCopySerialisedMap last, once all service maps have
```

Comment:
I'll update this last, once we have a few stable perf runs with the service maps changes only.